docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -1,19 +1,36 @@
 ---
 title: Debug Office Add-ins on iPad and Mac
 description: ''
-ms.date: 01/25/2018
+ms.date: 03/21/2018
 ---
 
 # Debug Office Add-ins on iPad and Mac
 
 You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on the iPad or Mac. Because add-ins are developed using HTML and Javascript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on an iPad or Mac. 
 
-## Debugging with Vorlon.JS 
+## Debugging with Safari Web Inspector on a Mac
```

Comment:
suggest changing "....on a Mac" to "...on Mac" (to be consistent with wording of the article title)

Reply:
Per style guide, don't use "with" as a synonym for using or by using. Suggest you change to "Debug using Vorlon.JS on a Mac". I beg to differ with Kim, per style guide it's "a Mac" singular, unless it's part of a product.

Reply:
the current topic indicates "on a Mac" so will stay with that per style guide. thanks